src/context/authContext.tsx

```
@@ -0,0 +1,41 @@
+interface AuthContextType {
+  authenticated: boolean
+  authenticatedUser?: AuthenticatedUser
+  handleLogin: (response: AuthenticationResponse) => void
+  handleLogout: () => void
+}
+
+interface AuthProviderProps {
+  children: React.ReactNode
+}
+
+export const AuthContext = createContext({} as AuthContextType);
```

ESLint

[error] Delete `;`

(prettier/prettier)